buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -54,6 +61,40 @@ public void apply(Project project) {
                 }
               });
     }
+    if (System.getenv().containsKey("METALAVA_BINARY_PATH")) {
+      String METALAVA_BINARY_PATH = System.getenv("METALAVA_BINARY_PATH");
+      AndroidSourceSet mainSourceSet = android.getSourceSets().getByName("main");
+      List<String> sourcePaths = new ArrayList<>();
+      for (File directory : mainSourceSet.getJava().getSrcDirs()) {
+        sourcePaths.add(directory.getAbsolutePath());
+      }
+      String sourcePathArgument = String.join(",", sourcePaths);
+      project.getTasks().create("apiInformation", ApiInformationTask.class, task -> {
```

Comment:
You should prefer `.register()` instead of `.create()` as it is evaluated lazily.